passageidentity/passage.py

```diff
@@ -111,7 +111,7 @@ def __fetchJWKS(self):
         r = requests.get(f"https://auth.passage.id/v1/apps/{self.app_id}/.well-known/jwks.json")
 
         if r.status_code != 200:
-            raise PassageError("Could not fetch JWKs for app id " + self.app_id)
+            raise PassageError("Could not fetch JWKs for app id " + self.app_id, r.status_code, r.reason, r.json()["error"])
```

Comment:
Should this be
```suggestion
            raise PassageError("Could not fetch JWKs for app id " + self.app_id, r.status_code, r.reason, r.json())
```
because the error value is being accessed here: https://github.com/passageidentity/passage-python/blob/e73533feabb94e825b31bf45fee32f1d3f06f180/passageidentity/errors.py#L10

Reply:
Great catch.  Thank you